.github/workflows/label-by-title.yml

```
@@ -0,0 +1,65 @@
+
+permissions:
+  issues: write
+  pull-requests: write
+  contents: read
+
+jobs:
+  labeling:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Label issue or PR based on title keywords
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)